src/JSON/JsonMapParser.hpp

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright_License {
+
+  XCSoar Glide Computer - http://www.xcsoar.org/
+  Copyright (C) 2000-2021 The XCSoar Project
+  A detailed list of copyright holders can be found in the file "AUTHORS".
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+
+#ifndef XCSOAR_CLOUD_JSONMAPPARSER_HPP
+#define XCSOAR_CLOUD_JSONMAPPARSER_HPP
+
+#undef Bool  // this makes trouble under LINUX
+#include <unordered_map>
+#include <string>
+
+typedef std::unordered_map<std::string, std::string> json_map_t;
```

Comment:
Why go through all the hoops to convert a JSON value to a map? This only adds overhead, both CPU-wise and memory-wise. Nothing of this `json_map_t` is JSON specific.

Reply:
But very fast for the Programmer ;-) I know, this isn't a good solution and I want to interpret the json text with your component from cm4all, but this needs much more time and I want to do this in a 2nd step. Because there is no usage in flight I hoped you can live with. One of my next PR should integrate your json component.

Reply:
The code you actually wrote is more complicated than just leaving it as `boost::json::object`. That conversion makes no sense - it's additional code *and* adds overhead. Your reply does not explain why this complication and overhead is justified.